app/views/auth_key_pair_cloud/_form.html.haml

```diff
@@ -2,8 +2,7 @@
   = _('Basic Information')
   %br
   %br
-
-.form-horizontal
+.form-horizontal{:id => 'start_form_div', :style => 'display:none',	'form-changed' => true}
```

Comment:
Pretty sure this is broken .. you just want `form-changed` .. but on the form ;)